src/geode/inspector/criterion/adjacency/surface_adjacency.cpp

```diff
@@ -117,12 +117,12 @@ namespace geode
             const auto adjacent_polygon =
                 mesh_.polygon_adjacent( polygon_edge );
             const auto polygon_adj_id = adjacent_polygon.value();
-            const auto v0 = mesh_.polygon_vertex( polygon_edge );
+            const auto v0 = mesh_.polygon_edge_vertex( polygon_edge, 0 );
```

Comment:
I remember that this has lower performance than `mesh_.polygon_vertex( polygon_edge );`

Reply:
True, but I found it a bit confusing, since it expects a PolygonVertex, and it then works using the fact that edges are numbered using their first vertex ... so is it durable or is it a hack ?